openmls/src/key_packages/mod.rs

```diff
@@ -108,31 +108,25 @@ impl KeyPackagePayload {
 pub struct KeyPackage {
     payload: KeyPackagePayload,
     signature: Signature,
-    encoded: Vec<u8>,
```

Comment:
@franziskuskiefer I think you introduced that at some point to reduce serialization operations. Can you remember how big the impact was?

Also: Should we maybe do this in a separate commit/PR?

Reply:
I ran benchmarks on both versions and there is no measurable difference (i.e. the tests are actually a tad faster after this change).